client/components/temp-drawer.jsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import { Drawer, IconButton, makeStyles, List, ListItem, Accordion, AccordionSummary, AccordionDetails, Typography, Link } from '@material-ui/core';
+import { createTheme, ThemeProvider } from '@material-ui/core/styles';
+import MenuIcon from '@material-ui/icons/Menu';
+
+const drawerWidth = 200;
+
+const useStyles = makeStyles(theme => {
+  return {
+    drawer: {
+      width: drawerWidth
+    },
+    drawerPaper: {
+      width: drawerWidth
+    },
+    positioning: {
+      position: 'absolute',
+      top: 20,
+      left: 10
+    },
+    listFont: {
+      fontFamily: 'Roboto',
+      padding: 0
+    }
+  };
+});
+
+export default function TempDrawer() {
+  const classes = useStyles();
+  const theme = createTheme({
+    overrides: {
+      MuiAccordion: {
+        root: {
+          backgroundColor: '#F0F5F9',
+          width: '100%'
+        }
+      },
+      MuiAccordionSummary: {
+        root: {
+          fontWeight: '500'
+        }
+      },
+      MuiAccordionDetails: {
+        root: {
+          backgroundColor: '#e8e8e8'
+        }
+      },
+      MuiDrawer: {
+        paper: {
+          backgroundColor: '#C9D6DF'
+        }
+      }
+    }
+  });
+
+  const [state, setState] = React.useState(false);
+  const toggleDrawer = e => {
+    state === true ? setState(false) : setState(true);
+  };
```

Comment:
How about `isDrawerOpen` and `setIsDrawerOpen` instead of just `state` and `setState`